Remove library CI tests that point to external sources.
These external sources did not have a permanent URL so are unreliable.
Tests can't simply be excluded since CMake will complain about missing sources.

Author: micketeer@gmail.com
--- cmake/testing/libmmg2d_tests.cmake.orig	2022-06-13 22:54:57.697924494 +0200
+++ cmake/testing/libmmg2d_tests.cmake	2022-06-13 22:56:27.167776567 +0200
@@ -36,7 +36,6 @@
   libmmg2d_ls_example0
   libmmg2d_lsOnly
   libmmg2d_lsAndMetric
-  test_api2d_0
   )
 SET ( MMG2D_LIB_TESTS_MAIN_PATH
   ${PROJECT_SOURCE_DIR}/libexamples/mmg2d/adaptation_example0/example0_a/main.c
@@ -47,7 +46,6 @@
   ${PROJECT_SOURCE_DIR}/libexamples/mmg2d/io_multisols_example0/main.c
   ${PROJECT_SOURCE_DIR}/libexamples/mmg2d/IsosurfDiscretization_lsOnly/main.c
   ${PROJECT_SOURCE_DIR}/libexamples/mmg2d/IsosurfDiscretization_lsAndMetric/main.c
-  ${MMG2D_CI_TESTS}/API_tests/2d.c
   )
 
 IF ( LIBMMG2D_STATIC )
@@ -69,7 +67,6 @@
     libmmg2d_fortran_io
     libmmg2d_fortran_lsOnly
     libmmg2d_fortran_lsAndMetric
-    test_api2d_fortran_0
     test_io2d_fortran
     )
 
@@ -79,7 +76,6 @@
     ${PROJECT_SOURCE_DIR}/libexamples/mmg2d/io_multisols_example0/main.F90
     ${PROJECT_SOURCE_DIR}/libexamples/mmg2d/IsosurfDiscretization_lsOnly/main.F90
     ${PROJECT_SOURCE_DIR}/libexamples/mmg2d/IsosurfDiscretization_lsAndMetric/main.F90
-    ${MMG2D_CI_TESTS}/API_tests/2d.F90
     ${PROJECT_SOURCE_DIR}/cmake/testing/code/mmg2d_io.F90
     )

--- cmake/modules/mmg2d.cmake.orig	2022-06-14 18:32:00.431611009 +0200
+++ cmake/modules/mmg2d.cmake	2022-06-14 18:31:57.852586956 +0200
@@ -220,10 +220,6 @@
         "${PROJECT_SOURCE_DIR}/libexamples/mmg2d/IsosurfDiscretization_lsOnly/multi-mat-sol.sol"
         "${CTEST_OUTPUT_DIR}/libmmg2d_lsAndMetric_multimat.o"
         )
-      ADD_TEST(NAME test_api2d_0   COMMAND ${TEST_API2D_EXEC0}
-        "${MMG2D_CI_TESTS}/API_tests/2dom.mesh"
-        "${CTEST_OUTPUT_DIR}/test_API2d.o"
-       )
 
 
       IF ( CMAKE_Fortran_COMPILER)
@@ -261,18 +257,6 @@
           "${PROJECT_SOURCE_DIR}/libexamples/mmg2d/IsosurfDiscretization_lsOnly/multi-mat-sol.sol"
           "${CTEST_OUTPUT_DIR}/libmmg2d_lsAndMetric_multimat.o"
           )
-        ADD_TEST(NAME test_api2d_fortran_0   COMMAND ${TEST_API2D_FORTRAN_EXEC0}
-          "${MMG2D_CI_TESTS}/API_tests/2dom.mesh"
-          "${CTEST_OUTPUT_DIR}/test_API2d.o"
-          )
-        ADD_TEST(NAME test_io2d_fortran_scalar   COMMAND ${TEST_IO2D_FORTRAN_EXEC}
-          "${MMG2D_CI_TESTS}/Hybrid/hybrid.mesh"
-          "${CTEST_OUTPUT_DIR}/hybrid-2d-scal.o" 0
-          )
-        ADD_TEST(NAME test_io2d_fortran_array   COMMAND ${TEST_IO2D_FORTRAN_EXEC}
-          "${MMG2D_CI_TESTS}/Hybrid/hybrid.mesh"
-          "${CTEST_OUTPUT_DIR}/hybrid-2d-array.o" 1
-          )
 
       ENDIF()

--- cmake/testing/libmmg3d_tests.cmake.orig	2022-06-14 19:26:45.958431873 +0200
+++ cmake/testing/libmmg3d_tests.cmake	2022-06-14 19:27:49.551026041 +0200
@@ -35,9 +35,6 @@
   libmmg3d_example6_io
   libmmg3d_lsOnly
   libmmg3d_lsAndMetric
-  test_api3d_0
-  test_api3d_domain-selection
-  test_api3d_vtk2mesh
   )
 SET ( MMG3D_LIB_TESTS_MAIN_PATH
   ${PROJECT_SOURCE_DIR}/libexamples/mmg3d/adaptation_example0/example0_a/main.c
@@ -48,9 +45,6 @@
   ${PROJECT_SOURCE_DIR}/libexamples/mmg3d/io_multisols_example6/main.c
   ${PROJECT_SOURCE_DIR}/libexamples/mmg3d/IsosurfDiscretization_lsOnly/main.c
   ${PROJECT_SOURCE_DIR}/libexamples/mmg3d/IsosurfDiscretization_lsAndMetric/main.c
-  ${MMG3D_CI_TESTS}/API_tests/3d.c
-  ${MMG3D_CI_TESTS}/API_tests/domain-selection.c
-  ${MMG3D_CI_TESTS}/API_tests/vtk2mesh.c
   )
 
 IF ( LIBMMG3D_STATIC )
@@ -72,7 +66,6 @@
     libmmg3d_fortran_io
     libmmg3d_fortran_lsOnly
     libmmg3d_fortran_lsAndMetric
-    test_api3d_fortran_0
     )
 
   SET ( MMG3D_LIB_TESTS_MAIN_PATH ${MMG3D_LIB_TESTS_MAIN_PATH}
@@ -81,7 +74,6 @@
     ${PROJECT_SOURCE_DIR}/libexamples/mmg3d/io_multisols_example6/main.F90
     ${PROJECT_SOURCE_DIR}/libexamples/mmg3d/IsosurfDiscretization_lsOnly/main.F90
     ${PROJECT_SOURCE_DIR}/libexamples/mmg3d/IsosurfDiscretization_lsAndMetric/main.F90
-    ${MMG3D_CI_TESTS}/API_tests/3d.F90
     )
 
 ENDIF ( CMAKE_Fortran_COMPILER )
--- cmake/modules/mmg3d.cmake.orig	2022-06-14 18:30:39.374855012 +0200
+++ cmake/modules/mmg3d.cmake	2022-06-14 19:22:51.535241582 +0200
@@ -248,26 +248,6 @@
         "${PROJECT_SOURCE_DIR}/libexamples/mmg3d/IsosurfDiscretization_lsOnly/m.sol"
         "${CTEST_OUTPUT_DIR}/libmmg3d_lsAndMetric_multimat.o"
         )
-      ADD_TEST(NAME test_api3d_0   COMMAND ${TEST_API3D_EXEC0}
-        "${MMG3D_CI_TESTS}/API_tests/2dom.mesh"
-        "${CTEST_OUTPUT_DIR}/test_API3d.o"
-       )
-      ADD_TEST(NAME test_api3d_domain-selection   COMMAND ${TEST_API3D_DOMSEL}
-        "${MMG3D_CI_TESTS}/OptLs_plane/plane.mesh"
-        "${MMG3D_CI_TESTS}/OptLs_plane/p.sol"
-        "${CTEST_OUTPUT_DIR}/test_API3d-domsel-whole.o"
-        "${CTEST_OUTPUT_DIR}/test_API3d-domsel-dom2.o"
-        )
-      ADD_TEST(NAME test_api3d_vtk2mesh   COMMAND ${TEST_API3D_VTK2MESH}
-        "${MMG3D_CI_TESTS}/API_tests/cellsAndNode-data.vtk"
-        "${CTEST_OUTPUT_DIR}/test_API3d-vtk2mesh.o"
-        )
-
-      IF ( NOT VTK_FOUND )
-        SET(expr "VTK library not founded")
-        SET_PROPERTY(TEST test_api3d_vtk2mesh
-          PROPERTY PASS_REGULAR_EXPRESSION "${expr}")
-      ENDIF ( )
 
       IF ( CMAKE_Fortran_COMPILER)
         SET(LIBMMG3D_EXECFORTRAN_a  ${EXECUTABLE_OUTPUT_PATH}/libmmg3d_fortran_a)
@@ -302,10 +282,6 @@
           "${PROJECT_SOURCE_DIR}/libexamples/mmg3d/IsosurfDiscretization_lsOnly/plane.mesh"
           "${PROJECT_SOURCE_DIR}/libexamples/mmg3d/IsosurfDiscretization_lsOnly/m.sol"
           "${CTEST_OUTPUT_DIR}/libmmg3d_lsAndMetric_multimat.o" )
-       ADD_TEST(NAME test_api3d_fortran_0   COMMAND ${TEST_API3D_FORTRAN_EXEC0}
-         "${MMG3D_CI_TESTS}/API_tests/2dom.mesh"
-         "${CTEST_OUTPUT_DIR}/test_API3d.o"
-         )
 
       ENDIF()
 
